### ISSUE TITLE
Purge for excess metadata

### ISSUE THREAD

**Issue by arybakov05** [interaktivgmbh/interaktiv.alttextgenerator#2]
When an alt text is modified, the metadata, such as the model used and the generation time, are not removed. There should be an option to remove such data.

This could be simplified by removing all metadata when modifying an alt text, rather than  only setting the AI generated flag to false.
